### PR TITLE
Integration test run

### PR DESCRIPTION
**Pull request by AxelVivien25**

### CODE REVIEW

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 1]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+
+    expected_tags = [RunTag(key="tag0", value="val2"),
+                     RunTag(key="tag1", value="test3"),
+                     RunTag(key="tag2", value="val2"),
+                     RunTag(key="tag3", value="test3")]
+
+    expected_run_data = RunData(metrics=expected_metrics,
+                                params=expected_params, tags=expected_tags)
+
+    expected_run = Run(run_info=expected_run_info, run_data=expected_run_data)
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:217]: You don't need to create these objects, you can directly check with the objects you use to create these

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 2]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+    assert actual_run._info == run._info
+    assert actual_run._data.__dict__ == run._data.__dict__
+
+
+@pytest.mark.usefixtures('init_store')
+def test_update_run_info(init_store):
+    run_info = init_store.update_run_info(
+        "d57a45f3763e4827b7c03f03d60dbbe1", RunStatus.FINISHED, 20)
+    actual_run = init_store.get_run("d57a45f3763e4827b7c03f03d60dbbe1")
+    assert run_info == actual_run._info
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:239]: Here you can check specifically the infos you changed

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 3]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+                                                           timestamp=1597324762890, step=2,
+                                                           is_nan=False),
+                                       ElasticLatestMetric(key="new_metric", value=7.0,
+                                                           timestamp=10, step=0, is_nan=False)]
+    init_store.log_metric("7b2e71956f3d4c08b042624a8d83700d", new_metric)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, latest_metric in enumerate(actual_run.latest_metrics):
+        assert latest_metric == expected_elastic_latest_metrics[i]
+    for i, metric in enumerate(actual_run.metrics):
+        assert metric == expected_elastic_metrics[i]
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:307]: I'm not sure to understand these checks, here you logged a single metric, so you should only check that this metric is present in the run data

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 4]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+                                       ElasticLatestMetric(key="new_metric", value=7.0,
+                                                           timestamp=10, step=0, is_nan=False),
+                                       ElasticLatestMetric(key="nan_metric", value=0.0,
+                                                           timestamp=10, step=0, is_nan=True)]
+    init_store.log_metric("7b2e71956f3d4c08b042624a8d83700d", new_metric)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, latest_metric in enumerate(actual_run.latest_metrics):
+        assert latest_metric == expected_elastic_latest_metrics[i]
+    for i, metric in enumerate(actual_run.metrics):
+        assert metric == expected_elastic_metrics[i]
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:344]: Same comment than for previous test, only check that you logged the metric and check its value

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 5]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+                                                           timestamp=10, step=0, is_nan=True),
+                                       ElasticLatestMetric(key="inf_metric",
+                                                           value=1.7976931348623157e308,
+                                                           timestamp=10, step=0, is_nan=False)]
+    init_store.log_metric("7b2e71956f3d4c08b042624a8d83700d", new_metric)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, latest_metric in enumerate(actual_run.latest_metrics):
+        assert latest_metric == expected_elastic_latest_metrics[i]
+    for i, metric in enumerate(actual_run.metrics):
+        assert metric == expected_elastic_metrics[i]
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:386]: Same than for NaN metric

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 6]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+                                                           timestamp=10, step=0, is_nan=False),
+                                       ElasticLatestMetric(key="negative_inf_metric",
+                                                           value=-1.7976931348623157e308,
+                                                           timestamp=10, step=0, is_nan=False)]
+    init_store.log_metric("7b2e71956f3d4c08b042624a8d83700d", new_metric)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, latest_metric in enumerate(actual_run.latest_metrics):
+        assert latest_metric == expected_elastic_latest_metrics[i]
+    for i, metric in enumerate(actual_run.metrics):
+        assert metric == expected_elastic_metrics[i]
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:435]: Same

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 7]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:485]: Here only check that you added this metric to the metrics array, and that you updated latest_metric value for this metric

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 8]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+    new_param = Param(key="new_param", value="new_value")
+    expected_elastic_params = [ElasticParam(key="param0", value="val2"),
+                               ElasticParam(key="param1", value="Val1"),
+                               ElasticParam(key="param2", value="Val1"),
+                               ElasticParam(key="param3", value="valeur4"),
+                               ElasticParam(key="new_param", value="new_value")]
+    init_store.log_param("7b2e71956f3d4c08b042624a8d83700d", new_param)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, param in enumerate(actual_run.params):
+        assert param == expected_elastic_params[i]
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:499]: Only check that the param you added is present in the run data

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 9]
```diff
@@ -219,3 +180,343 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
...
+    new_tag = RunTag(key="new_tag", value="new_value")
+    expected_elastic_tags = [ElasticTag(key="tag0", value="val2"),
+                             ElasticTag(key="tag1", value="test3"),
+                             ElasticTag(key="tag2", value="val2"),
+                             ElasticTag(key="tag3", value="test3"),
+                             ElasticTag(key="new_tag", value="new_value")]
+    init_store.set_tag("7b2e71956f3d4c08b042624a8d83700d", new_tag)
+    actual_run = init_store._get_run("7b2e71956f3d4c08b042624a8d83700d")
+    for i, tag in enumerate(actual_run.tags):
+        assert tag == expected_elastic_tags[i]
```
Comment (t-henri) [inline tests/integration_tests/test_integration_elasticsearchstore.py:513]: Same than for param

File: tests/integration_tests/test_integration_elasticsearchstore.py [hunk 10]
```diff
@@ -209,6 +346,8 @@ def test_list_all_columns_deleted(init_store):
                                params=["param7"],
                                tags=["tag7"])
     actual_columns = init_store.list_all_columns("hjb553MBNoOYfhXjp3Tn", ViewType.DELETED_ONLY)
+    print(actual_columns)
+    print(expected_columns)
```
Comment (jdlesage) [inline tests/integration_tests/test_integration_elasticsearchstore.py:350]:
```suggestion
```

print statements to be removed

**Pull request merged**